### PR TITLE
mashup project - My album

### PR DESCRIPTION
**Pull request by msqintian**

### CODE REVIEW

**Comment by msqintian** [comment]
the assignment in NYU class page is closed. I cannot submit the link of the pull request.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+In this project, I called APIs from flickrs and google. The goal of this project is to grab my albums
+stored in flickrs, and list all photos in each album. When I show these photos to my friends, I can 
+click 'show on map' to show them where these pictures were taken. It is also nice to see the map of 
+my footprints.
```
Comment (afeld) [inline README.md:4]: Really like that you wrote this!  Thanks for the inspiration: https://github.com/advanced-js/mashup/commit/ff30d48fb3032845f67e810b63ab37c2bc8b29b6

File: flickr.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+};
+
+//class methods
+App.Album.showAll = function(data){
+  var jsonData = xmlToJson(data);
+  var response = jsonData.rsp.photosets;
+  var totalAlbums = response['@attributes'].total;
+  $('#albumNumber').html(totalAlbums);
+
+  if (totalAlbums > 0){
```
Comment (afeld) [inline flickr.js:32]: Is this line necessary?

File: flickr.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    $('#'+this.id).append("<table></table>");
+    if(totalPhotos == 1){
+      eachPhoto = response.photo['@attributes']; 
+      this.displayEachPhoto(eachPhoto);
+    }else{
+      for(var i=0; i<totalPhotos; i++){
+        eachPhoto = response.photo[i]['@attributes']; 
+        this.displayEachPhoto(eachPhoto);
+      }
+    }
```
Comment (afeld) [inline flickr.js:91]: Probably worth adding a comment as to why it needs to be handled both ways.

**Comment by afeld** [comment]
Creating an album on [the live site](http://msqintian.github.io/mashup/) didn't work for me, but your code is very clear and well-written.  Awesome job!

**Comment by afeld** [comment]
Also, no worries about submitting to Classes, I got it entered for ya.

**Pull request closed without merging**